assets/components/contributionSelection/__tests__/__snapshots__/contributionSelectionReducerTest.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`Contributions Selection reducer should not change the state for an action with the wrong scope 1`] = `
 Object {
-  "annualAmount": "75",
+  "annualAmount": "50",
```

Comment:
could we put these at the bottom of this structure, with `oneOffAmount` and `monthlyAmount`?

Reply:
This .snap file is generated running `yarn jest-update-snapshot` so don't need to move these out to values.

Reply:
👍